test-zeekoe.py

```diff
@@ -18,6 +18,8 @@
 import subprocess
 import sys
 import random
+import requests
```

Comment:
`requests` is not built-in so please add a comment at the top to install via pip.

Reply:
done!